File: Backend/src/main/java/com/mycompany/myapp/service/dto/VentaAsientosResponse.java
package com.mycompany.myapp.service.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.List;
@Data
@AllArgsConstructor
@NoArgsConstructor
public class VentaAsientosResponse {
    private Long eventoId;
    private Long ventaId;
    private LocalDateTime fechaVenta;
    private List<AsientosVentasDTO> asientos;
    private Boolean resultado;
    private String descripcion;
    private Double precioVenta;
}

File: Frontend/composeApp/src/commonMain/kotlin/org/example/project/dto/VentaAsientosResponse.kt
package org.example.project.dto
import kotlinx.serialization.Serializable
import kotlin.time.Instant

@Serializable
data class VentaAsientosResponse (
    val eventoId: Long,
    val ventaId: Long ?= null,
    val fechaVenta: String,
    val asientos: List<AsientosVentasDTO>,
    val resultado: Boolean,
    val descripcion: String,
    val precioVenta: Double
)